tests/unit/test_auth.py

```diff
@@ -36,7 +36,7 @@ def test_open_spotify_session(mocker):
     # Assert that the SpotifyOAuth was called with correct parameters
     mock_spotify_oauth.assert_called_once_with(
         username="test_user",
-        scope="playlist-read-private",
+        scope="playlist-read-private, user-library-read",
```

Comment:
Probably would be better to store the scopes in a variable in the auth module and access those from here instead of re-defining

Reply:
yes that would be a good approach, i will look into it and make the required changes